packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IProgressIndicatorProps {
   percentComplete?: number;
 
   /**
+   * Text alternative of progress status
```

Comment:
May want to clarify the description here. Something like:

"Text alternative of the progress status, used by screen readers for reading the value of the progress."

I don't really understand the use cases here to be honest, so we need the docs to be specific. Why as a consumer should I care about this?

According to the spec, it really doesn't sound like this is a good use case.

"Authors SHOULD only set the aria-valuetext attribute when the rendered value cannot be accurately represented as a number. For example, a slider may have rendered values of small, medium, and large. In this case, the values of aria-valuenow could range from 1 through 3, which indicate the position of each value in the value space, but thearia-valuetext would be one of the strings: small, medium, or large."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute